lava_test_plans/testcases/mmtests-blogbench.yaml

```diff
@@ -0,0 +1,5 @@
+{% extends "testcases/master/template-mmtests.yaml.jinja2" %}
+
+{% set config_file = 'configs/config-io-blogbench' %}
```

Comment:
Can't we just stick with the config_file name for the name of the file too?
config_file|replace('configs/config-', 'mmtests-').

Makes it easier?
so the test_name and the name of the file match.

That's how it is today in tuxrun too.

Reply:
We can do the following:
```jinja2
{# We use template's file name to calculate config name to keep them in sync #}
{% set config_file = 'configs/' + self._TemplateReference__context.name|replace('mmtests', 'config')|replace('.yaml','') %}
```